imagetest/test_suites/oslogin/setup.go

```diff
@@ -14,7 +14,7 @@ func TestSetup(t *imagetest.TestWorkflow) error {
 		return err
 	}
 	vm.AddMetadata("enable-oslogin", "true")
-	vm.RunTests("TestOsLoginEnabled")
+	vm.RunTests("TestOsLoginEnabled|TestGetentPasswdAllUsers|TestGetPasswdOsloginUser|TestGetPasswdOsloginUID|TestGetentPasswdLocalUser|TestGetentPasswdInvalidUser")
```

Comment:
totally optional feedback but these are prefixes, so you could just say `TestGetent` here if you don't need to split them across hosts.